languages/rust/oso/src/host/class.rs

```diff
@@ -175,6 +175,20 @@ where
         self
     }
 
+    pub fn add_values_method<F, Args, I>(mut self, name: &str, f: F) -> Self
```

Comment:
`add_iterator_method`? With a docstring indicating the generation of choices behavior.

Reply:
:+1: